src/probnum/filtsmooth/gaussfiltsmooth/kalmanposterior.py

```diff
@@ -162,56 +173,6 @@ def sample(self, locations=None, size=()):
             [self.sample(locations=locations, size=size[1:]) for _ in range(size[0])]
         )
 
-    def _single_sample_path(self, locations, random_vars):
-
-        # Make sure the final rv is informed about all the data points.
-        # Either condition on the final RV (a single smoothing step) and sample
-        # or (if last element in random_vars is final rv) sample directly.
-        if locations[-1] < self.locations[-1]:
-            t, rv = locations[-1], random_vars[-1]
-
-            # Intermediate prediction
-            predicted_rv, info = self.gauss_filter.predict(
-                rv=rv,
-                start=t,
-                stop=self.locations[-1],
-            )
-            crosscov = info["crosscov"]
```

Comment:
turns out this function was not called anymore after #316  (which is why us playing around in here yesterday did not change anything).

Reply:
:D that explains why things still worked